msg/actuator_controls_saturation.msg

```diff
@@ -0,0 +1,16 @@
+uint64 timestamp  # time since system start (microseconds)
+
+uint16 bitmask    # bitmask of the flags below
+bool valid        # 0 - true when the saturation status is used
+bool roll_pos     # 1 - true when a positive roll demand change will increase saturation
```

Comment:
What about making this slightly closer to the form it's actually consumed in?

https://github.com/PX4/PX4-Autopilot/blob/f55590ce78e97213aad6e3f3c2e20b3b47523d23/src/modules/mc_rate_control/RateControl/RateControl.cpp#L52-L57

Would this work?

``` C
bool[3] torque_saturation_positive
bool[3] torque_saturation_negative

bool[3] thrust_saturation_positive
bool[3] thrust_saturation_negative
```

You could also skip the bitmask and valid flag. Don't even publish if it's not valid.

Reply:
I'm thinking about going a step further and sending the saturation level (value) and not just a boolean. I'll push a proposal

Reply:
see https://github.com/PX4/PX4-Autopilot/pull/18565#issuecomment-963238178

Reply:
This is now largely redundant with control_allocator_status, albeit the actuator_output saturation information.

Reply:
> This is now largely redundant with control_allocator_status, albeit the actuator_output saturation information.

I completely skipped that this message existed. Let me remove `actuator_controls_saturation` and simply use `control_allocator_status` then (and add `control_allocator_status` to the logged topics).